test/fernet_spec_SUITE.erl

```diff
@@ -0,0 +1,81 @@
+-module(fernet_spec_SUITE).
+-include_lib("eunit/include/eunit.hrl").
+-include_lib("common_test/include/ct.hrl").
+-compile(export_all).
+-define(EPOCH_OFFSET, 62167219200).
+
+%% Specific test cases or groups to run.  The test case is named as a
+%% single atom.  Groups are named as {group, GroupName}.  The tests
+%% will run in the order given in the list.
+all() ->
+    [verify_test
+    ,invalid_test
+    ,generate_test].
+
+%%====================================================================
+%% Setup and teardown
+%%====================================================================
+
+init_per_suite(Config) ->
+    Config.
+
+end_per_suite(Config) ->
+    Config.
+
+%% Runs before the test case.  Runs in the same process.
+init_per_testcase(_CaseName, Config) ->
+    Config.
+
+%% Runs after the test case.  Runs in the same process.
+end_per_testcase(_CaseName, Config) ->
+    Config.
```

Comment:
You can drop all the init functions if you don't use them. they don't need to be exported nor defined.